tap/tlstapper/tls_poller.go

```diff
@@ -161,6 +156,28 @@ func (p *tlsPoller) handleTlsChunk(chunk *tlsTapperTlsChunk, extension *api.Exte
 	return nil
 }
 
+func (p *tlsPoller) getAddressPair(chunk *tlsTapperTlsChunk) (addressPair, error) {
```

Comment:
This function should use `getKprobeAddressPair`, no fallbacks, and remove the old sockFd and addressFd

Reply:
Somewhat fixed cf6598b8e5f6eecb568c7566122cba75cdf8d3db.
Didn't remove but improved the code. I think it is more straightforward now.
Couldn't remove because the Go TLS tapper still fetches the addresses from sock fd.